/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
/ceph/home/yixin01/xiaoxuan/code/AlphaChimp_private/mmtracking/mmtrack/core/utils/misc.py:26: UserWarning: Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed.
  warnings.warn(
/ceph/home/yixin01/xiaoxuan/code/AlphaChimp_private/mmtracking/mmtrack/core/utils/misc.py:36: UserWarning: Setting MKL_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed.
  warnings.warn(
loading annotations into memory...
Done (t=0.50s)
creating index...
index created!
Traceback (most recent call last):
  File "tools/evaluate_tracking.py", line 192, in <module>
    main()
  File "tools/evaluate_tracking.py", line 182, in main
    metric = dataset.evaluate(outputs, **eval_kwargs)
  File "/ceph/home/yixin01/xiaoxuan/code/AlphaChimp_private/mmtracking/mmtrack/datasets/leipzigchimp_mot_dataset.py", line 344, in evaluate
    raise KeyError(f'metric {metric} is not supported.')
KeyError: 'metric bbox track is not supported.'
/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/torch/distributed/launch.py:178: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torch.distributed.run.
Note that --use_env is set by default in torch.distributed.run.
If your script expects `--local_rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  warnings.warn(
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 1961234) of binary: /ceph/home/yixin01/anaconda3/envs/alphachimp_track/bin/python
Traceback (most recent call last):
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/torch/distributed/launch.py", line 193, in <module>
    main()
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/torch/distributed/launch.py", line 189, in main
    launch(args)
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/torch/distributed/launch.py", line 174, in launch
    run(args)
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/torch/distributed/run.py", line 689, in run
    elastic_launch(
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 116, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/ceph/home/yixin01/anaconda3/envs/alphachimp_track/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 244, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
***************************************
   tools/evaluate_tracking.py FAILED   
=======================================
Root Cause:
[0]:
  time: 2024-10-23_15:30:54
  rank: 0 (local_rank: 0)
  exitcode: 1 (pid: 1961234)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
=======================================
Other Failures:
  <NO_OTHER_FAILURES>
***************************************

